docs/specs/Broker-CI.md

```diff
@@ -0,0 +1,103 @@
+# Broker CI
+
+The Broker CI spec will outline how the project is going create CI.
+
+## Introduction
+The Broker CI will involve multiple projects: ansible-service-broker
+and apb-examples repo. The goal is to outline the possible CI for
+each repo to achieve a successful CI run outlined in the next section.
+
+## Successful CI Run
+A successful CI run is spawning the broker and using the latest
+rhscl-postgresapb and mediawiki-apb images to build a MediaWiki page.
+
+## Environment
+The expected running service in the environment in all test cases will be:
+- Docker
+- Openshift Cluster
+- Service-Catalog
```

Comment:
I'd link to the service-catalog project:

[Service-Catalog](https://github.com/kubernetes-incubator/service-catalog) since it's not a well known project like Docker and OpenShift.